.gitignore

```diff
@@ -6,6 +6,9 @@ MANIFEST
 __pycache__/
 *.py[cod]
 
+# Also don't include .pyc filse
```

Comment:
These files is already covered by line 7

Reply:
I will remove that line.